docs/apis-tools/spring-zeebe-sdk/configuration.md

```diff
@@ -298,21 +298,51 @@ public ProcessVariables foo(@VariablesAsType ProcessVariables variables, @Custom
 }
 ```
 
-### Throwing `CamundaBpmnError`s
+### Throwing `BpmnError`s
 
-Whenever your code hits a problem that should lead to a [BPMN error](/components/modeler/bpmn/error-events/error-events.md) being raised, you can throw a `CamundaBpmnError` to provide the error code used in BPMN:
+Whenever your code hits a problem that should lead to a [BPMN error](/components/modeler/bpmn/error-events/error-events.md) being raised, you can throw a `BpmnError` to provide the error code used in BPMN:
 
 ```java
 @JobWorker(type = "foo")
 public void handleJobFoo() {
   // some work
-  if (!successful) {
+  if (businessError) {
    // problem shall be indicated to the process:
-   throw new CamundaBpmnError("DOESNT_WORK", "This does not work because...");
+   throw CamundaError.bpmnError("ERROR_CODE", "Some explanation why this does not work");
+   // this is a static function that returns an instance of BpmnError
   }
 }
 ```
 
+### Failing jobs in a controlled way
+
+Whenever you want a job to fail in a controlled way, you can throw a `JobError` and provide parameters like `variables`, `retries` and `retryBackoff`:
+
+```java
+@JobWorker(type = "foo")
+public void handleJobFoo() {
+  try {
+   // some work
+  } catch(Exception e) {
+   // problem shall be indicated to the process:
+   throw CamundaError.jobError("Error message", new ErrorVariables(), null, Duration.ofSeconds(10), e);
+   // this is a static function that returns an instance of JobError
+  }
+}
+```
+
+The JobError takes 5 parameters:
+
+- `errorMessage`: String
+- `variables`: Object _(optional)_, default `null`
+- `retries`: Integer _(optional)_, defaults to `job.getRetries() - 1`
+- `retryBackoff`: Duration _(optional)_, defaults to `PT0S`
+- `cause`: Exception _(optional)_, defaults to `null`
+
+:::note
+When the job error is sent to the engine, the stacktrace of the job error will become the actual error message. The provided cause will be visible in Operate.
```

Comment:
🔧  I would be in favor of actually making transparent that the failJob API is used.
```suggestion
The job error is sent to the engine by the SDK calling the [Fail Job API](/apis-tools/camunda-api-rest/specifications/fail-job.api.mdx). The stacktrace of the job error will become the actual error message. The provided cause will be visible in Operate.
```